### PR TITLE
Fix README example

### PR DESCRIPTION
**Pull request by JasmineLowen**
Hey cool crate! Just found out about it the other day on mastodon.

I'll try to use it for some things I want to learn that use `winit` in the examples. So far it seems really cool and promising!

It took me a little while to figure out how to use the crate though, since the README example has some typos and that's where I generally look before looking into the example directory.

In this PR I fixed the second README example and made sure it compiles.

Thanks for the crate again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,6 @@ fn main2(evl: EventLoop<()>) {
 }
 
 fn main() {
-#   return;
```
Comment (notgull) [inline README.md]: This is necessary for the tests to work without opening up a window, so I'd like to avoid removing it.

**Review by notgull — CHANGES_REQUESTED** [review]
Aw crap, I forgot to add the CI pipeline, didn't I?

Looks good to me aside from one small thing.

File: README.md (hunk 1 above)
Comment (JasmineLowen) [reply README.md]: Ahh, I see. Sure! Added

**Review by notgull — APPROVED** [review]
Thank you!

**Pull request merged**